web/styles/zulip.css

```diff
@@ -2194,21 +2194,32 @@ body:not(.spectator-view) {
             top: 0.2419em;
         }
 
-        .dropdown-list-delete {
+        .dropdown-list-delete,
+        .dropdown-list-edit {
             position: absolute;
             top: 0;
             right: 5px;
             visibility: hidden;
         }
 
+        .dropdown-list-edit {
+            color: var(--color-text-neutral-icon-button);
+            right: 30px;
+
+            &:hover {
+                color: var(--color-text-neutral-icon-button-hover);
+            }
+        }
```

Comment:
We should be setting the `intent` of `icon-button` to `neutral` instead of manually setting the colors here.

Reply:
Refer https://github.com/zulip/zulip/pull/33902/commits/b31024be47a4bede541c0576100978b5d91e5381#r1994205616.